docs/tools.md

```diff
@@ -0,0 +1,50 @@
+# Tools
+
+## Foundry
+
+You can verify contracts using Foundry in Hedera using our verification service.
+Our verification service instance is located at `https://server-verify.hashscan.io`.
```

Comment:
I think we are not supposed to mention specific deployment sites. 
We can use placeholders instead:  `<hedera-sourcify server URL>`, `<hedera-json-rpc-relay URL>`...

Reply:
I see. The goal here is to enable developers to use our deployed instance as well. @Nana-EC might be able to shed some light here.

Reply:
Correct, usually this is the case 
We are okay mentioning Hedera instances. The Hashscan Explorer is one case where we hold back since there all multiple options for users to manage.
In this case the smart contract verification hosted by Swirldslabs domain is being used for the Hedera ecosystem to tie into.

If there's a way to allow users to enter the endpoint that'd be the preferred option.
If not whiles we're establishing integrations for the ecosystem and there's onlyb1 option for the ecosystem we can apply an exception in this case.

Reply:
Great, I'll go ahead and merge this then.